openid-4-verifiable-presentations-1_0.md

```diff
@@ -581,12 +581,24 @@ The following is a non-normative example of the Authorization Response that is s
     state=eyJhb...6-sVA
 ```
 
-If the Response Endpoint has successfully processed the request, it MUST respond with HTTPS status code 200. 
+The following is a non-normative example of an Authorization Error Response that is sent as an HTTP POST request to the Verifier's Response Endpoint:
+
+```
+  POST /post HTTP/1.1
+    Host: client.example.org
+    Content-Type: application/x-www-form-urlencoded
```

Comment:
```suggestion
POST /post HTTP/1.1
Host: client.example.org
Content-Type: application/x-www-form-urlencoded
```

Reply:
Applied your suggestion. I had to fix that that line indentation in other places too.